clcache.py

```diff
@@ -206,7 +206,7 @@ def computeKey(compilerBinary, commandLine):
         (preprocessedSourceCode, pperr) = preprocessor.communicate()
 
         if preprocessor.returncode != 0:
-            sys.stderr.write(pperr)
+            sys.stderr.write(pperr.decode("utf-8"))
```

Comment:
I see. What do you think about this implementation? https://github.com/frerich/clcache/pull/117/commits/5fd8a4142418ac4c27db3548fe9e8f3b8e44b82c